demoji/__init__.py

```diff
@@ -217,3 +217,42 @@ def replace(string, repl=""):
     """
     set_emoji_pattern()
     return _EMOJI_PAT.sub(repl, string)
+
+
+def findall_list(string):
+    """Find emojis within ``string`` like findall() but returns a list with each occurrence instead of dictionary. 
+    Useful for looping through. Added by David Agudelo (@davidagud)
+
+    :param string: The input text to search
+    :type string: str
+    :return: A list of ``[emoji: description]``
```

Comment:
The list comp below looks to return a list of ``[description]``, not ``[emoji: description]``